framework/wazuh/tests/test_util.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python
+# Copyright (C) 2015-2019, Wazuh Inc.
+# Created by Wazuh, Inc. <info@wazuh.com>.
+# This program is a free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+
+import pytest
+import os
+
+from wazuh.utils import WazuhVersion
+
+test_data_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'data')
+
+
+@pytest.mark.parametrize('version1, version2', [
+    ('Wazuh v3.5.0', 'Wazuh v3.5.2'),
+    ('Wazuh v3.6.1', 'Wazuh v3.6.3'),
+    ('Wazuh v3.7.2', 'Wazuh v3.8.0'),
+    ('Wazuh v3.8.0', 'Wazuh v3.8.1'),
+    ('Wazuh v3.9.0', 'Wazuh v3.9.2'),
+    ('Wazuh v3.9.10', 'Wazuh v3.9.14'),
+    ('Wazuh v3.10.1', 'Wazuh v3.10.10'),
+    ('Wazuh v4.10.10', 'Wazuh v4.11.0'),
+    ('Wazuh v5.1.15', 'Wazuh v5.2.0')
```

Comment:
I miss more examples. What about versions without the 'Wazuh ' prefix?

Reply:
I added more unit tests:

```bash
# /var/ossec/framework/python/bin/pytest -s framework/wazuh/tests/test_util.py -vv
==================================================== test session starts ====================================================
platform linux -- Python 3.7.2, pytest-4.5.0, py-1.8.0, pluggy-0.11.0 -- /var/ossec/framework/python/bin/python3.7
cachedir: .pytest_cache
rootdir: /root/wazuh-old/wazuh/framework
collected 24 items                                                                                                          

framework/wazuh/tests/test_util.py::test_version_ok[Wazuh v3.5.0-Wazuh v3.5.2] PASSED
framework/wazuh/tests/test_util.py::test_version_ok[Wazuh v3.6.1-Wazuh v3.6.3] PASSED
framework/wazuh/tests/test_util.py::test_version_ok[Wazuh v3.7.2-Wazuh v3.8.0] PASSED
framework/wazuh/tests/test_util.py::test_version_ok[Wazuh v3.8.0-Wazuh v3.8.1] PASSED
framework/wazuh/tests/test_util.py::test_version_ok[Wazuh v3.9.0-Wazuh v3.9.2] PASSED
framework/wazuh/tests/test_util.py::test_version_ok[Wazuh v3.9.10-Wazuh v3.9.14] PASSED
framework/wazuh/tests/test_util.py::test_version_ok[Wazuh v3.10.1-Wazuh v3.10.10] PASSED
framework/wazuh/tests/test_util.py::test_version_ok[Wazuh v4.10.10-Wazuh v4.11.0] PASSED
framework/wazuh/tests/test_util.py::test_version_ok[Wazuh v5.1.15-Wazuh v5.2.0] PASSED
framework/wazuh/tests/test_util.py::test_version_ok[v3.6.0-v3.6.1] PASSED
framework/wazuh/tests/test_util.py::test_version_ok[v3.9.1-v3.9.2] PASSED
framework/wazuh/tests/test_util.py::test_version_ok[v4.0.0-v4.0.1] PASSED
framework/wazuh/tests/test_util.py::test_version_ok[3.6.0-3.6.1] PASSED
framework/wazuh/tests/test_util.py::test_version_ok[3.9.0-3.9.2] PASSED
framework/wazuh/tests/test_util.py::test_version_ok[4.0.0-4.0.1] PASSED
framework/wazuh/tests/test_util.py::test_version_ko[v3.6.0-v.3.6.1] PASSED
framework/wazuh/tests/test_util.py::test_version_ko[Wazuh v4-Wazuh v5] PASSED
framework/wazuh/tests/test_util.py::test_version_ko[Wazuh v3.9-Wazuh v3.10] PASSED
framework/wazuh/tests/test_util.py::test_version_ko[ABC v3.10.1-ABC v3.10.12] PASSED
framework/wazuh/tests/test_util.py::test_version_ko[Wazuhv3.9.0-Wazuhv3.9.2] PASSED
framework/wazuh/tests/test_util.py::test_version_ko[3.9-3.10] PASSED
framework/wazuh/tests/test_util.py::test_version_ko[3.9.0-3.10] PASSED
framework/wazuh/tests/test_util.py::test_version_ko[3.10-4.2] PASSED
framework/wazuh/tests/test_util.py::test_version_ko[3-3.9.1] PASSED
```